motor/core.py

```diff
@@ -2034,3 +2034,10 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     async def get_keys(self):
         cursor_class = create_class_with_framework(AgnosticCursor, self._framework, self.__module__)
         return cursor_class(self.delegate.get_keys(), self)
+
+    async def create_encrypted_collection(self, *args, **kwargs):
+        collection_class = create_class_with_framework(
+            AgnosticCollection, self._framework, self.__module__
+        )
+        coll, ef = await self.delegate.create_encrypted_collection(*args, **kwargs)
```

Comment:
self.delegate is a pymongo ClientEncryption so we can't use await here.

Reply:
Done.